src/router.js

```diff
@@ -1,85 +1,73 @@
-            meta: {layout: 'empty-layout'},
-            component: () => import('./views/murren/SignUp.vue')
-        },
-        {
-            path: '/murren',
-            name: 'murren',
-            meta: {layout: 'main-layout', accessTokenExpected: true},
-            component: () => import('./views/murren/Murren.vue')
-        },
-        {
-            path: '/check_email',
-            name: 'check_email',
-            meta: {layout: 'main-layout'},
-            component: () => import('./views/murren/CheckEmail.vue')
-        },
-        {
-            path: '/murren_email_activate',
-            name: 'murren_email_activate',
-            meta: {layout: 'main-layout'},
-            component: () => import('./views/murren/ConfirmEmail.vue')
-        },
-        {
-            path: '/set_new_password',
-            name: 'set_new_password',
-            meta: {layout: 'main-layout'},
-            component: () => import('./views/murren/SetNewPassword.vue')
-        },
-    ]
+  mode: 'history',
+  routes: [
+    {
+      path: '/',
+      name: 'home',
+      meta: {layout: 'main-layout', murr_some_data: '42'},
+      component: () => import('./views/Home.vue'),
+    },
+    {
+      path: '/login',
+      name: 'login',
+      meta: {layout: 'empty-layout'},
+      component: () => import('./views/murren/Login.vue'),
+    },
+    {
+      path: '/signup',
+      name: 'signup',
+      meta: {layout: 'empty-layout'},
+      component: () => import('./views/murren/SignUp.vue'),
+    },
+    {
+      path: '/murren',
+      name: 'murren',
+      meta: {layout: 'main-layout', accessTokenExpected: true},
+      component: () => import('./views/murren/Murren.vue'),
+    },
+    {
+      path: '/check_email',
+      name: 'check_email',
+      meta: {layout: 'main-layout'},
+      component: () => import('./views/murren/CheckEmail.vue'),
+    },
+    {
+      path: '/murren_email_activate',
+      name: 'murren_email_activate',
+      meta: {layout: 'main-layout'},
+      component: () => import('./views/murren/ConfirmEmail.vue'),
+    },
+    {
+      path: '/set_new_password',
+      name: 'set_new_password',
+      meta: {layout: 'main-layout'},
+      component: () => import('./views/murren/SetNewPassword.vue'),
+    },
+  ],
 });
 
-import {Message} from 'element-ui';
-
 router.beforeEach((to, from, next) => {
 
-    if (to.matched.some(record => record.meta.accessTokenExpected)) {
-
-        if (!store.getters.accessToken_getters) {
-
-            store.dispatch('changeShowLoginForm_actions');
-
-            Message({
-                message: 'Требуется авторизация 😳',
-                center: true,
-                customClass: 'element-ui-message__error',
-                type: 'warning',
-                showClose: true
-            });
+  /* All Successful */
+  if ((to.matched.some(r => r.meta.accessTokenExpected) && store.state.auth.accessToken) ||
+    !to.matched.some(r => r.meta.accessTokenExpected)) {
```

Comment:
why you check !to.matched.some? in first section of || we check all accessTokenExpected situation

Reply:
**!to.matched.some** is needed to trigger the first check in the current code and
when using the boolean operator **||** order is not important

p.s. maybe I somehow misunderstood the question?